pkg/tfvars/libvirt/cache.go

```diff
@@ -45,12 +43,7 @@ func cachedImage(uri string) (string, error) {
 		return uri, err
 	}
 
-	cache := diskcache.NewWithDiskv(diskv.New(diskv.Options{
-		BasePath:     httpCacheDir,
```

Comment:
Optional extra: Detect when the old directory exists and warn that we don't use it anymore.

Reply:
> Optional extra: Detect when the old directory exists and warn that we don't use it anymore.

Ah yes, and in fact we’re still creating it here even though we don’t need it...